src/Bicep.LangServer/Handlers/BicepDeploymentParametersHandler.cs

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
```

Comment:
If I have *any* object parameters in the bicep file (I assume arrays as well), you don't ask for *any* parameter files, is that intentional?
```bicep
param b string
param c string
param a object
```
```json
{
    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
    "contentVersion": "1.0.0.0",
    "parameters": {
    }
}
```

Reply:
It goes straight to deployment without asking for param values (and of course fails on 'c', but should it be asking for the others, too, or showing an error in the UI?)

Reply:
Yes, this is intentional. We won't support object and arrays(atleast in v1). We expect params of type array/object to either have a default value or specified in parameters file.
We ask for param values in this case:
```
param b string
param c string
param a object = {
  displayName: 'test'
  description: 'test'
}
```
This would fail:
```
param b string
param c string
param a object
```
With error message:
![image](https://user-images.githubusercontent.com/30270536/171867452-0912fdf7-b73b-4a60-8740-d4b23b96b922.png)

If you are not seeing the error message, I think it's because you are missing this commit I made yesterday:
https://github.com/Azure/bicep/commit/27c2b1e8f26f8bcaad48f2290a7db62d2867e93b

Let me know if you have any concerns here. Thanks!

Reply:
I guess we'll wait for feedback